### ISSUE TITLE
Multi-label retraining

### ISSUE THREAD

**Issue by pawel12** [tensorflow/hub#71]
Hi,
Would you be so kind for all community and add an example of multi-label image classification retraining?
Thanks in advance :)

**Comment by akhorlin** [comment]
We will look into adding such example. Meanwhile, you should be able to
modify existing retrain.py
<retrain.pyhttps://github.com/tensorflow/hub/blob/master/examples/image_retraining/retrain.py>
example to use tf.losses.sigmoid_cross_entropy
<tf.losses.sigmoid_cross_entropyhttps://www.tensorflow.org/api_docs/python/tf/losses/sigmoid_cross_entropy>
as the loss in order to perform multi-label image classification.

**Comment by psonthalia** [comment]
I am very new to TensorFlow, and I would love to be able to do this. Do you happen to have some starter code to be able to do this? Or would it require a large change? Thanks!

**Comment by nanshihui** [comment]
any update?
